tools/reviewer/reviewer.proto

```diff
@@ -25,9 +25,18 @@ message CIRequest {
 message CIResponse {
   CIRequest request = 1;
   message TargetResult {
+    enum Status {
+      NONE = 0;
+      SUCCESS = 1;
+      FAIL = 2;
+      RUNNING = 3;
+      OUTDATED = 4;
+    }
     CIRequest.Target target = 1;
-    bool success = 2;
-    string log = 3;
+    Status status = 2;
+    // Log weighs too much
+    // TODO: move log somewhere else
+    // string log = 3;
```

Comment:
It's ok, you can leave the `log` here, we just won't fill it up.
We;ll use the same proto and store the log somewhere else in Firestore inside it.